apis/fluentd/v1alpha1/fluentd_types.go

```diff
@@ -41,9 +41,9 @@ type FluentdSpec struct {
 	// Select cluster output plugins used to send all logs that did not match any route to the matching outputs
 	DefaultOutputSelector *metav1.LabelSelector `json:"defaultOutputSelector,omitempty"`
 	// By default will build the related service according to the globalinputs definition.
-	DisableService bool `json:"disableService,omitempty"`
-	// Numbers of the Fluentd instance
-	Replicas *int32 `json:"replicas,omitempty"`
+	DisableService  bool            `json:"disableService,omitempty"`
```

Comment:
It is not a good idea to delete the `replicas` field directly, we should set it to deprecated and delete it in the future.

Reply:
I have reverted this change.